lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -48,7 +48,7 @@
 
     &:hover {
       .nx-radio-checkbox__control {
-        border-color: var(--nx-color-form-element-border);
+        border-color: var(--nx-swatch-grey-05);
         filter: none;
         stroke: var(--nx-swatch-grey-05);
```

Comment:
I don't think border and stroke are actually necessary here at all; they just repeat what's already present via the non-focus `:hover` selector

Reply:
> I don't think border and stroke are actually necessary here at all; they just repeat what's already present via the non-focus `:hover` selector

Thanks! Removed here:
5c5ea6780bb688265db64fa897b7852cd8c5935e